xml/deployment_configuration.xml

```diff
@@ -129,7 +129,9 @@
     <filename>cloud.cfg</filename>, but you have to configure network first,
     e.g. by using local configuration files. The url is specified during boot
     time and must be in the format:
-    <literal>cloud-init-url=http://<replaceable>hostname.domain</replaceable>/cloud.cfg</literal>.
+    <literal>
```

Comment:
As discussed on IRC, please remove the changes in line 132-134, otherwise you will revert what I already fixed in 247757572.